contracts/CrossChain.sol

```diff
@@ -487,6 +490,74 @@ contract CrossChain is Config, Initializable, ICrossChain {
         emit ParamChange(key, value);
     }
 
+    function handleAckPackageFromMultiMessage(
+        bytes calldata _multiMessagePayload,
+        uint8 _packageType,
+        uint64 _multiMessageSequence
+    ) external whenNotSuspended onlyMultiMessage {
+        uint8 _channelId = uint8(_multiMessagePayload[0]);
+        if (_multiMessagePayload.length < 33) {
```

Comment:
let us do the check before we assign the `_channelId`

Reply:
fixed